scalecube-organization/src/main/java/io/scalecube/tokens/JwtApiKey.java

```diff
@@ -40,7 +40,7 @@ public static Builder builder() {
     private Map<String, String> claims;
     private String id;
     private String name;
-    private long tokenTimeToLiveInMillis;
+    private Long tokenTimeToLiveInMillis;

```

Comment:
should be max long value else set to something else

Reply:
Why it should be `Long.MAX_VALUE`?

If we want a token that never expires we should not add `exp` claim to a token instead of setting it as `Long.MAX_VALUE`.